### PR TITLE
Hidden content rule fixes

### PR DESCRIPTION
**Pull request by marcysutton**
There were a couple of issues with the new hidden content rules when I pulled them into the extensions.

### CODE REVIEW

**Comment by WilcoFiers** [comment]
Good point, although, who puts aria-hidden on a script tag.

File: lib/checks/visibility/hidden-content.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 let styles = window.getComputedStyle(node);
 
-if (axe.commons.dom.hasContent(node)) {
+let whitelist = ['SCRIPT', 'HEAD', 'TITLE', 'NOSCRIPT', 'STYLE'];
+if (!whitelist.includes(node.tagName) && axe.commons.dom.hasContent(node)) {
```
Comment (WilcoFiers) [inline lib/checks/visibility/hidden-content.js]: tagName should be cast to uppercase to have it work with xhtml docs.

**Comment by downzer0** [comment]
What if you change the selector in the rule to `body *`?

File: lib/checks/visibility/hidden-content.js (hunk 1 above)
Comment (dylanb) [inline lib/checks/visibility/hidden-content.js]: Add template to this list

**Comment by marcysutton** [comment]
I wanted to include HTML in case somebody did something really bad.

**Comment by downzer0** [comment]
👍

**Pull request merged**